internal/fs/fs.go

```diff
@@ -211,3 +211,13 @@ func CopyFolder(origFolderPath, destFolderPath string) error {
 
 	return nil
 }
+
+func TestWrite(dir string) error {
+	tempFile, err := os.CreateTemp(dir, "drandtestwrite-")
```

Comment:
I'm confused as to why go has a `CreateTemp` func other than setting the dir to `/tmp` if the first param is `""`

Reply:
Yeah it does that:
> If dir is the empty string, CreateTemp uses the default directory for temporary files, as returned by TempDir.